pygotham/frontend/templates/profile/settings.html

```diff
@@ -18,6 +18,14 @@
         {{ horizontal_field(form.name) }}
         {{ horizontal_field(form.email) }}
         {{ horizontal_field(form.bio) }}
+        <div class="row">
+          <div class="large-4 columns">
+            <label>
+              Twitter Handle
+              {{ form.twitter_handle }}
```

Comment:
I figured since twitter handles are short, they should get a short field. I will keep the `horizontal_field` in the new PR.